saleor/graphql/product/types/products.py

```diff
@@ -842,14 +868,19 @@ def get_node(cls, info, id):
             return qs.filter(id=id).first()
         return None
 
+    @staticmethod
+    def resolve_channel_listing(root: ChannelContext[models.Collection], _info):
```

Comment:
Lack of permissions.

Reply:
Still unresolved. Non-staff users shouldn't be able to get any od channel_listing.